playbooks/byo/config.yml

```diff
@@ -1,2 +1,14 @@
 ---
+- name: Verify Requirements
```

Comment:
I think @mtnbikenc was working on some graphical representation of what calls what... that would be useful to look at now. Any pointers to this dependency graph somewhere?